### PR TITLE
Added an integration test for sharing /dev/shm in a pod.

### PR DESCRIPTION
**Pull request by qianzhangxa**
## High-level description

What features does this change enable? What bugs does this change fix?

Add an integration test for sharing /dev/shm in a pod.

## Corresponding DC/OS tickets (required)

  - [DCOS-54620](https://jira.mesosphere.com/browse/DCOS-54620) Add an integration test for sharing /dev/shm in a pod.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

File: packages/dcos-integration-test/extra/test_ucr.py [hunk 1]
```diff
@@ -348,3 +348,38 @@ def test_if_ucr_pods_can_be_deployed_with_non_root_user_ephemeral_volume(dcos_ap
...
+                'resources': {'cpus': 0.1, 'mem': 32},
+                'image': {'kind': 'DOCKER', 'id': 'library/alpine'},
+                'exec': {'command': {'shell': 'df -m /dev/shm | grep -w 1234'}},
+                'linuxInfo': {'ipcInfo': {'mode': 'SHARE_PARENT'}}
+            },
+            {
+                'name': 'container2',
+                'resources': {'cpus': 0.1, 'mem': 32},
+                'exec': {'command': {'shell': 'echo $PING > foo; while true; do sleep 1; done'}},
+                'healthcheck': {'command': {'shell': 'test $PING = `cat foo`'}}
```
Comment (vinodkone) [inline packages/dcos-integration-test/extra/test_ucr.py]: do you also want to check that this container doesnt get access to the shm?

**Comment by qianzhangxa** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by qianzhangxa** [comment]
@mesosphere-mergebot changelog-not-required Added an integration test

File: packages/dcos-integration-test/extra/test_ucr.py (hunk 1 above)
Comment (abudnik) [reply packages/dcos-integration-test/extra/test_ucr.py]: how about
```
'exec': {'command': {'shell': 'test -z "$(df -m /dev/shm | grep -w 1234)" && echo $PING > foo; while true; do sleep 1; done'}},
```

Comment (Gilbert88) [reply packages/dcos-integration-test/extra/test_ucr.py]: > do you also want to check that this container doesnt get access to the shm?

We don't have this case. Containers may:
1. have their own shm
2. share from parent
3. share from the agent

https://github.com/apache/mesos/blob/master/include/mesos/mesos.proto#L3301

Comment (Gilbert88) [reply packages/dcos-integration-test/extra/test_ucr.py]: @qianzhangxa As Andrei commented above, we probably need to verify the size is actually 1234 by test -z 'df -m /dev/shm | grep -w 1234'

Comment (qianzhangxa) [reply packages/dcos-integration-test/extra/test_ucr.py]: Done.

**Comment by Gilbert88** [comment]
@mesosphere-mergebot label Ship It

**Pull request merged**